rasa/utils/tensorflow/model_data_utils.py

```diff
@@ -415,9 +410,10 @@ def _extract_features(
                 list_of_features = zero_features

```

Comment:
on line before, do we need to update `attribute_mask[i] = 0` due to additional dim?